mailet/standard/src/test/java/org/apache/james/transport/mailets/ReplaceContentTest.java

```diff
@@ -129,8 +79,8 @@ public void serviceShouldReplaceSubjectWhenMatching() throws Exception {
     @Test
     public void serviceShouldReplaceBodyWhenMatching() throws Exception {
         mailetConfig.setProperty("subjectPattern", "/prova/PROVA/i/,/a/e//,/o/o/i/");
-        mailetConfig.setProperty("bodyPattern", "/prova/PROVA/i/," + "/a/e//,"
-                + "/o/o/i/,/\\u00E8/e'//," + "/prova([^\\/]*?)ble/X$1Y/im/,"
+        mailetConfig.setProperty("bodyPattern", "/prova/PROVA/i/," + "/a/e/r/,"
+                + "/O/o/r/,/\\u00E8/e'//," + "/prove([^\\/]*?)ble/X$1Y/im/,"
```

Comment:
Thank you for the translations. You have some remaining prova(ble) in this test.